HikVisionSource/README.md

```diff
@@ -17,7 +17,7 @@ has been split into two parts, [Setting Up Your Machine](#machine) and [Setting
 
 # Prerequisites <a name="pre" id="pre"></a>
 
-In order to compile correctly, you must download the Hikvision SDK from [*here*](https://www.hikvision.com/en/support/download/sdk/), choosing the Device Network SDK (for Linux 64 bit).
+In order to compile correctly, you must download the Hikvision SDK from [*here*](https://www.hikvision.com/en/support/download/sdk/), choosing the Device Network SDK (Select the correct OS you are using : for Windows of for Linux 64 bit).
 For windows based system, you should download the Device Network SDK (for Windoes 64 bit) as well from the same location.
```

Comment:
can probably delete this line since you include Windows above.  (and Windoes is a typo)
```suggestion
For windows based system, you should download the Device Network SDK (for Windoes 64 bit) as well from the same location.
```

Reply:
done